src/OrchardCore/OrchardCore.Admin.Abstractions/Models/AdminSettings.cs

```diff
@@ -4,6 +4,8 @@ namespace OrchardCore.Admin.Models;
 
 public class AdminSettings
 {
+    public const string AdminMenuId = "adminMenu";
```

Comment:
This is not really a setting, a separate const class would be better.

Reply:
moved it